### DISCUSSION TITLE
snippets within strings?

### DISCUSSION THREAD

**Discussion by johnnywz00** [toerob/vscode-tads3tools#11] [Ideas]
Is it possible for snippets to be activated while typing within a string? Typically the completions menu (including available snippets) only pops up when typing in a source code context, but I'd find it really helpful if I could get a 
<<one of>>$1<<or>>$2<<stopping>>$0
type of snippet within a string. Even if the typical completions dialog doesn't pop up within a string, could there be a keyboard shortcut to bring up the snippet?

**Comment by toerob** [comment]
It's definitely worth digging into!

**Reply by johnnywz00** [reply]
Sorry, I didn't realize about the formatting there.  I was trying to do a "one of"$1"or"$2"stopping"$0 construction
